src/vmq_queue.erl

```diff
@@ -296,6 +275,13 @@ drain(Event, State) ->
     lager:error("got unknown event in drain state ~p", [Event]),
     {next_state, drain, State}.
 
+drain({enqueue_many, Msgs}, _From, #state{drain_over_timer=TRef} =  State) ->
+    %% if multiple queues trigger queue migration at the 'same' time
+    %% so we've to queue those message otherwise they would be lost.
```

Comment:
Hey @dergraf, can you elaborate a bit on this comment, why would the messages be lost in case multiple queues trigger queue migration against a queue at the same time? I'm not sure I understand the comment compared with the code which basically just allows the `enqueue_many` while in the `drain` state.